wolox-books/src/app/register/register.component.html

```diff
@@ -0,0 +1,34 @@
+<form [formGroup]="rForm" (ngSubmit)="addPost(rForm.value)" class="form">
+  <div class="form-container">
+    <header class="header">
+      <img src="assets/wolox_logo.svg" class="header-logo" alt="Wolox logo">
+      <h1 class="header-title">BOOKS</h1>
+    </header>
+    <div class="form-signup">
+      <label class="form-field">First name
+        <input type="text" formControlName="firstName" class="form-input">
+      </label>
+      <p class="alert" *ngIf="!rForm.controls['firstName'].valid && rForm.controls['firstName'].touched">{{ fieldRequiredAlert }}</p>
+      <label class="form-field">Last name
+        <input type="text" formControlName="lastName" class="form-input">
+      </label>
+      <p class="alert" *ngIf="!rForm.controls['lastName'].valid && rForm.controls['lastName'].touched">{{ fieldRequiredAlert }}</p>
+      <label class="form-field">Email
+        <input type="text" formControlName="email" class="form-input">
```

Comment:
If the field is for an email, then the `input` should be of type email instead of text